src/System.Collections/tests/Performance/Perf.List.cs

```diff
@@ -186,6 +186,21 @@ public void Enumerator(int size)
         [InlineData(1000)]
         [InlineData(10000)]
         [InlineData(100000)]
+        public void SetCapacity(int size)
+        {
+            List<object> list = CreateList(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < 10000; i++)
+                    {
+                        list.Capacity = size + 1;
```

Comment:
This is measuring the performance of Capacity when it doesn't actually need to change the size of the list, since it's always being set to the same value.  Is that intentional?